projects/solar/inc/sense.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+// Generic sense module.
+// Requires interrupts, soft timers, and the event queue (for data_store) to be initialized.
```

Comment:
What I usually do is I specify all the transitive dependencies of a module in the initialization requirements comment so that whoever's using the module doesn't have to root through all of the dependencies. In this case it's kind of weird since there isn't actually a `data_store_init` function, so it would confusing to say "requires data_store to be initialized"; it's probably best to just say "requires the event queue to be initialized" without specifying it's for data_store.

Reply:
ah that's a good point, I think there aren't all that many transitive dependencies in our codebase so I hadn't thought about it before.